lib/web_ui/lib/src/engine/dom.dart

```diff
@@ -1390,6 +1392,106 @@ extension DomCSSRuleListExtension on DomCSSRuleList {
   external int get length;
 }
 
+/// Trusted Types API
+
+/*
+// Trusted Types API (TrustedTypePolicy, TrustedScript, TrustedScriptURL)
+// https://developer.mozilla.org/en-US/docs/Web/API/TrustedTypesAPI
+*/
+
+/// A factory to create `TrustedTypePolicy` objects.
+@JS()
+@staticInterop
+abstract class DomTrustedTypePolicyFactory {}
+
+/// (Some) methods of the [DomTrustedTypePolicyFactory]:
+extension DomTrustedTypePolicyFactoryExtension on DomTrustedTypePolicyFactory {
+  /// createPolicy
+  external DomTrustedTypePolicy createPolicy(
+    String policyName,
+    DomTrustedTypePolicyOptions? policyOptions,
+  );
+}
+
+/// Options to create a trusted type policy.
+@JS()
+@staticInterop
+@anonymous
+abstract class DomTrustedTypePolicyOptions {
+  /// Constructs a TrustedPolicyOptions object in JavaScript.
```

Comment:
According to https://w3c.github.io/trusted-types/dist/spec/#trusted-type-policy-options the "dictionary" is referred to as `TrustedTypePolicyOptions`.

Reply:
Total typo! Luckily I got the name of the actual class right :P